fala/settings/base.py

```diff
@@ -142,6 +142,8 @@ def root(*x):
 
 LAALAA_API_HOST = os.environ.get("LAALAA_API_HOST", None)
 
+ENVIRONMENT = os.environ.get("ENVIRONMENT", "dev")
```

Comment:
I don't know if it was me or you, but this got reverted to "dev", I think you had it as "unknown" earlier, which is correct